botocore/__init__.py

```diff
@@ -44,10 +45,6 @@ def emit(self, record):
     ('DescribeStorediSCSIVolumes', '-'): 'describe-stored-iscsi-volumes',
     ('CreateStorediSCSIVolume', '_'): 'create_stored_iscsi_volume',
     ('CreateStorediSCSIVolume', '-'): 'create-stored-iscsi-volume',
```

Comment:
Let's address these as separate PRs if needed.  iSCI I think is too special cased to have a keyword for, ElastiCache isn't a concern for us (not used for opname/inputs), but Ec2 I believe we'll need to fix for opsworks.RegisterVolume.